keps/sig-release/2853-k-core-branch-rename/kep.yaml

```
@@ -14,9 +14,11 @@ creation-date: 2021-11-19
 reviewers:
   - "@saschagrunert"
   - "@justaugustus"
+  - "@BenTheElder"
 approvers:
   - "@saschagrunert"
   - "@justaugustus"
+  - "@BenTheElder"
 
 status: provisional
```

```suggestion
status: implementable
```

Beside of that I think we should update the KEP to reflect the latest timeline, like:

https://github.com/kubernetes/enhancements/blob/42a4b1c8a18780cf1d5c2460113d59b246002548/keps/sig-release/2853-k-core-branch-rename/README.md?plain=1#L212

To get the timings planned ahead, we should aim to get approval from SIG Testing / Architecture as well. Then we can start with the actual plan outlining everything in a note to k-dev.